docs/examples/federated_statistics.rst

```diff
@@ -6,5 +6,5 @@ Federated Statistics
 
    federated_statistics_overview
    Federated Statistics for medical imaging (Github) <https://github.com/NVIDIA/NVFlare/tree/main/examples/federated_statistics/image_stats/README.md>
-   Federated Statistics for tabular data with DataFrame (Github) <https://github.com/NVIDIA/NVFlare/tree/main/examples/federated_statistics/df_stats/README.md>
+   Federated Statistics for tabular data with DataFrame (Github) <https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/federated_statistics/df_stats/README.md>
```

Comment:
note that it is "federated-statistics" (minus instead of underscore)

https://github.com/NVIDIA/NVFlare/blob/main/examples/advanced/federated-statistics/df_stats/README.md

Reply:
Good catch. Fixed.